portfolio/src/main/java/com/google/sps/data/analysis/SentimentAnalysis.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.data.analysis;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.sps.common.Constants;
+import java.io.IOException;
+
+public final class SentimentAnalysis {
+
+  public enum TextEvaluation { POSITIVE, NEUTRAL, NEGATIVE; }
+
+  /**
+   * @return the feeling detected by Cloud Natural Language API,
+   *         when querying a comment from the website.
+   */
+  public static TextEvaluation evaluate(String text) throws IOException {
+
```

Comment:
nit: the style guide provides not much guidance here, but the empty line isn't helping me to understand code better.